pybwa/libbwaaln.pyi

```diff
@@ -55,5 +57,20 @@ class BwaAln:
     ) -> List[AlignedSegment]: ...
     def reinitialize_seed(self) -> None: ...
 
+@dataclass(frozen=True)
+class XaHit:
+    refname: str
+    start: int
+    negative: bool
+    cigar: Cigar
+    edits: int
+    md: str | None = None
+    rest: str | None = None
+    @property
+    def mismatches(self) -> int: ...
+    @property
+    def end(self) -> int: ...
+
+def to_xa_hits(rec: AlignedSegment | str | bytes) -> list[XaHit]: ...
```

Comment:
How would you feel about this function taking a `max_hits: Optional[int]` and returning only the first `max_hits` if the tag contains more?